libraries/lib-time-track/TimeTrack.cpp

```diff
@@ -222,7 +222,8 @@ void TimeTrack::Paste(double t, const Track &src)
       (void)0;// intentionally do nothing.
 }
 
-void TimeTrack::Silence(double WXUNUSED(t0), double WXUNUSED(t1))
+void TimeTrack::Silence(
+   double WXUNUSED(t0), double WXUNUSED(t1), ProgressReporter)
```

Comment:
I repeat, you can clean up the old `WXUNUSED` macros

Just `double, double,` or you can use C++17 `[[maybe_unused]]`